src/routes/graphql/index.ts

```diff
@@ -12,8 +18,17 @@ const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
         200: gqlResponseSchema,
       },
     },
-    async handler(req) {
-      return {};
+    async handler(req: FastifyRequest<{ Body: GqlRequestBody }>) {
+      const { query, variables } = req.body;
+
+      const result = await graphql({
+        schema,
+        source: query,
+        variableValues: variables,
+        contextValue: { fastify: fastify, prisma: fastify.prisma },
```

Comment:
prisma client as the context should be enough, no need in the whole fastify instance.